### ISSUE TITLE
nitpick: repo name

### ISSUE THREAD

**Issue by amueller** [jupyter-widgets/tutorial#206]
Feel free to ignore this nitpick, but sometimes it's handy to have self-contained repo names.

If someone `git clone`s the tutorial repo, they'll get a folder called ``tutorial``, which is not the best way to distinguish it from the other tutorials someone might do at scipy :)

**Comment by marimeireles** [comment]
It's a good point! I'd be a +1 to rename it `jupyter-widgets-tutorial` or any other name you think it's best?
@mwcraig @ianhi @ibdafna @bsyouness  what do you think about it?

**Comment by maartenbreddels** [comment]
I agree it would be good to rename, I'd say 'ipywidgets-tutorial'

**Comment by mwcraig** [comment]
I agree it should be renamed. I'm a little reluctant to call it ipywidgets-tutorial since we cover quite a few thing outside of the core ipywidgets. What about ipywidgets-ecosystem-tutorial or jupyter-widget-ecosystem-tutorial?

**Comment by bsyouness** [comment]
I agree! "The Jupyter Interactive Widget Ecosystem" is what the SciPy tutorial has been referred to as since at least 2017. So, I think it would make sense to call it "jupyter-widget-ecosystem-tutorial".
